examples/custom_azs.tf

```diff
@@ -4,14 +4,14 @@ provider "aws" {
 }
 
 module "vpc" {
-  source = "git@github.com:rackspace-infrastructure-automation/aws-terraform-vpc_basenetwork//?ref=v0.0.9"
+  source = "git@github.com:rackspace-infrastructure-automation/aws-terraform-vpc_basenetwork//?ref=v0.0.10"
 
   vpc_name   = "MyVPC"
   custom_azs = ["us-west-2a", "us-west-2b"]
 
-  cidr_range             = "10.0.0.0/16"
-  public_cidr_ranges     = ["10.0.1.0/24", "10.0.3.0/24"]
+  cidr_range             = "10.0.0.0/19"
+  public_cidr_ranges     = ["10.0.4.0/22", "10.0.8.0/22"]
   public_subnets_per_az  = 1
-  private_cidr_ranges    = ["10.0.2.0/24", "10.0.4.0/24"]
+  private_cidr_ranges    = ["10.0.12.0/22", "10.0.16.0/22"]
   private_subnets_per_az = 1
```

Comment:
why not leave the 10. with the broader range  to demonstrate  making a bigger range

Reply:
Just matching the CFn tests.